<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>REAL TIME MEASUREMENTS</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLCDNumber" name="lcdNumber_BUS_VOLTAGE">
             <property name="styleSheet">
              <string notr="true">  background-color: black
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>BUS VOLTAGE (V)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLCDNumber" name="lcdNumber_LOAD_VOLTAGE">
             <property name="styleSheet">
              <string notr="true">background:black;
foreground:green;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>LOAD VOLTAGE (V)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLCDNumber" name="lcdNumber_SHUNT_VOLTAGE">
             <property name="styleSheet">
              <string notr="true">background: black</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>SHUNT VOLTAGE (mV)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLCDNumber" name="lcdNumber_CURRENT">
             <property name="styleSheet">
              <string notr="true">background :black</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>CURRENT (mA)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLCDNumber" name="lcdNumber_POWER">
             <property name="styleSheet">
              <string notr="true"> background-color: black;

</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>POWER (mW)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_SERIAL_PORTS"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>START ACQUISITION</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>REAL TIME GRAPHS</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>BUS VOLTAGE</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>MINIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_BUS_VOLTAGE_MIN">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>MAXIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_BUS_VOLTAGE_MAX">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>14</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>LOAD VOLTAGE</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>MINIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_LOAD_VOLTAGE_MIN">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>MAXIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_LOAD_VOLTAGE_MAX">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>SHUNT VOLTAGE</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>MINIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_SHUNT_VOLTAGE_MIN">
               <property name="minimum">
                <number>-320</number>
               </property>
               <property name="maximum">
                <number>320</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>MAXIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_SHUNT_VOLTAGE_MAX">
               <property name="minimum">
                <number>-320</number>
               </property>
               <property name="maximum">
                <number>320</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>CURRENT</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>MINIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_CURRENT_MIN">
               <property name="minimum">
                <number>-320</number>
               </property>
               <property name="maximum">
                <number>320</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>MAXIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_CURRENT_MAX">
               <property name="minimum">
                <number>-320</number>
               </property>
               <property name="maximum">
                <number>320</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>POWER</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>MINIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_POWER_MIN">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>MAXIMUM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_POWER_MAX">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBothSides</enum>
               </property>
               <property name="tickInterval">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionExport_Samples_as_CSV"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport_Samples_as_CSV">
   <property name="text">
    <string>Export Samples</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
